development-vm/Pinfile

```diff
@@ -19,7 +19,8 @@ process :'contacts-admin' => [:'publishing-api', :whitehall]
 process :'content-audit-tool' => [:'publishing-api-read', :'content-audit-tool-sidekiq-google-analytics', :'content-audit-tool-sidekiq-publishing-api']
 process :'content-data-admin' =>[:'content-performance-manager']
 process :'content-performance-manager' => [:'publishing-api', :'content-performance-manager-sidekiq-default', :'content-performance-manager-publishing-api-consumer', 'content-performance-manager-bulk-import-publishing-api-consumer', 'content-store']
-process :'content-publisher' => [:'publishing-api', :'asset-manager']
+process :'content-publisher' => [:'publishing-api', :'asset-manager', :'content-publisher-sidekiq']
+process :'content-publisher-sidekiq'
```

Comment:
As per the comment on the process, I'm not sure we need this.

But if we do we'd expect it to have a dependency on publishing_api too.

Reply:
Will remove